app/controllers/unattended_controller.rb

```diff
@@ -67,6 +67,12 @@ def pxe_config
   # Using alias_method causes test failures as iPXE method is unknown in an empty DB
   def gPXE; iPXE; end
 
+  protected
+
+  def require_ssl?
```

Comment:
I honestly do not know it worked so far...
It seems that all of the actions require non-ssl connection